### ISSUE TITLE
How do denotational, axiomatic and operational semantics relate to statics and dynamics in PFPL?

### ISSUE THREAD

**Issue by CMCDragonkai** [CommBank/PLATYPUS#4]
Question in the title ^!

**Comment by thsutton** [comment]
I think we see denotational and operational semantics in PFPL in the big-step and small-step dynamics described in the chapters, no?

**Comment by CMCDragonkai** [comment]
What pages are you referring to?

**Comment by thsutton** [comment]
Although I suppose "this other element of the same inductively defined set of objects" probably isn't what you mean by denotational semantics.

**Comment by thsutton** [comment]
Right, I'm crazy. What we're doing as "dynamics" is operational semantics.

**Comment by SamRoberts** [comment]
Denotational still seems clearly to be related to dynamics. And operational semantics basically is dynamics afaict. Axiomatic semantics is not obvious to me though: that pre/post stuff looks an awful lot like what a type system does.

**Comment by thsutton** [comment]
I'll try to form a more informed opinion tonight.

**Comment by CMCDragonkai** [comment]
@thsutton Have you formulated your informed opinion yet?

**Comment by thsutton** [comment]
No
